docker/Dockerfile

```diff
@@ -45,6 +57,7 @@ RUN chmod +x /snap/bin/snapcraft
 FROM ubuntu:$UBUNTU
 COPY --from=builder /snap/core /snap/core
 COPY --from=builder /snap/core18 /snap/core18
```

Comment:
you can remove core18 (previous base for snapcraft)

Reply:
Hi @sergiusens,

thanks for the feedback! Just a question: Are you sure that core18 can safely be removed? My understanding is, that the image has to contain any core base that a potential downstream app might want to build against, correct? In this case, even if snapcraft itself wouldn't need the core18 base anymore, apps that build against it would start to fail. Or do I miss something here...?